vllm/__init__.py

```diff
@@ -33,3 +34,8 @@
     "initialize_ray_cluster",
     "PoolingParams",
 ]
+
+
+def githash():
+    import torch
+    return torch.ops._C.githash()
```

Comment:
I don't think this will work unless `vllm._C` is also imported.

Reply:
Let me test this. When filing the PR for vllm, I had to use torch.ops other than vllm._C. Now that we merged upstream torch_bindings.cpp etc files and deleted our pybind.cpp so I'm not sure. I'll manually verify again.

Reply:
I meant something like this:
```
import torch
import vllm._C  # this registers the new ops
return torch.ops._C.githash()
```

Reply:
oh i see, I tried:
```
    import torch
    return torch.ops._C.githash()
```
and it looks it's working:

Reply:
I think if you tried to `from vllm import githash` then it would not work without having the `import vllm._C`.

If you import all of vllm, you'll pick up all the extensions via `_custom_ops.py`

Reply:
I tried above but it looks also working:

```
>>> from vllm import githash
>>> githash()
'ec1715a452635d36348752f918bdb6f77cdf0bbe'
```

Is it because I'm testing on a cpu machine? Will it behave differently if I test on a gpu?

Reply:
Ok, I'm a bit surprised that works.  Maybe since `githash()` is defined in `init.py` it pulls in the stuff it needs somehow.

Reply:
Tried again on a GPU and it looks also working:
```
>>> from vllm import githash
>>> githash()
'ec1715a452635d36348752f918bdb6f77cdf0bbe'
```

Reply:
@bnellnm can you please confirm if I land this PR based on the findings above? Thanks.

Reply:
If it works I guess it's fine.

Reply:
Ok, thanks Bill!